client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/URLClassReplacement.java

```diff
@@ -69,7 +70,8 @@ public static URLConnection openConnection(URL caller) throws java.io.IOExceptio
                 URL newURL = new URL(url);
                 return newURL.openConnection();
             } else {
-                throw new UnknownHostException("There is no WireMock initiated for this hostname");
+                // Removed the exception to avoid restarts inside flaky
+                SimpleLogger.uniqueWarn("There is no WireMock initiated for this hostname");
```

Comment:
a warning is for unexpected behavior. having some tests in which wiremock is not up is expected, can comment out this warning

Reply:
in the generated tests, to represent these cases (without bytecode instrumentation and without connecting to actual host), the host should resolve to a loopback address which is not in use